shared/base/src/main/java/com/ivy/base/legacy/Transaction.kt

```diff
@@ -29,7 +29,7 @@ data class Transaction(
     val date: LocalDate? = null,
     val time: LocalTime? = null,
     val recurringRuleId: UUID? = null,
-
+    val paidFor : LocalDateTime? = null,/** to store the date for which the payment was made. */
```

Comment:
KDoc must be above `paidFor`

Reply:
Yeah have done that while checking for detekt